src/AElf.Kernel.Consensus.AEDPoS/AssemblyInfo.cs

```diff
@@ -3,3 +3,5 @@
 [assembly: InternalsVisibleTo("AElf.Kernel.Tests")]
 [assembly: InternalsVisibleTo("AElf.Contracts.Consensus.AEDPoS.Tests")]
 [assembly: InternalsVisibleTo("AElf.Contracts.Economic.TestBase")]
+[assembly: InternalsVisibleTo("AElf.Contracts.TestKit.AEDPoSExtension")]
```

Comment:
do not use it except test projects

Reply:
> do not use it except test projects

This is a project act as test base, like testkit.